dnf/cli/commands/updateinfo.py

```diff
@@ -280,8 +279,13 @@ def inst2mark(inst):
             return ('' if not mixed else 'i ' if inst else '  ')
 
         def type2label(typ, sev):
-            return (cls.SECURITY2LABEL[sev] if typ == hawkey.ADVISORY_SECURITY
-                    else cls.TYPE2LABEL[typ])
+            if typ == hawkey.ADVISORY_SECURITY:
+                if sev in cls.SECURITY2LABEL:
+                    return cls.SECURITY2LABEL.get(sev)
+                else:
+                    return _('Unknown/Sec.')
+            else:
+                return cls.TYPE2LABEL[typ]
```

Comment:
What happens if `typ` is not in `cls.TYPE2LABEL` or is `None`?

Reply:
Fixed.